tests/meta/upload/scout/conftest.py

```diff
@@ -0,0 +1,355 @@
+"""Fixtures for the upload scout api tests"""
+
+import json
+import logging
+from datetime import datetime
+from pathlib import Path
+from typing import List
+
+import pytest
+from housekeeper.store import models as hk_models
+
+from cg.constants import Pipeline
+from cg.meta.upload.scout.balsamic_config_builder import BalsamicConfigBuilder
+from cg.meta.upload.scout.mip_config_builder import MipConfigBuilder
+from cg.meta.upload.scout.scout_load_config import MipLoadConfig
+from cg.meta.upload.scout.scoutapi import UploadScoutAPI
+from cg.store import Store, models
+
+# Mocks
+from tests.mocks.hk_mock import MockHousekeeperAPI
+from tests.mocks.madeline import MockMadelineAPI
+from tests.mocks.scout import MockScoutAPI
+from tests.store_helpers import StoreHelpers
+
+LOG = logging.getLogger(__name__)
+
+
+class MockAnalysis:
+    """Mock an analysis object"""
+
+    @staticmethod
+    def get_latest_metadata(family_id=None):
+        """Mock get_latest_metadata"""
+        # Returns: dict: parsed data
+        # Define output dict
+        outdata = {
+            "analysis_sex": {"ADM1": "female", "ADM2": "female", "ADM3": "female"},
+            "family": family_id or "yellowhog",
+            "duplicates": {"ADM1": 13.525, "ADM2": 12.525, "ADM3": 14.525},
+            "genome_build": "hg19",
+            "rank_model_version": "1.18",
+            "mapped_reads": {"ADM1": 98.8, "ADM2": 99.8, "ADM3": 97.8},
+            "mip_version": "v4.0.20",
+            "sample_ids": ["2018-20203", "2018-20204"],
+            "sv_rank_model_version": "1.08",
+        }
+        return outdata
+
+    @staticmethod
+    def convert_panels(customer_id, panels):
+        """Mock convert_panels"""
+        _ = customer_id, panels
+        return ""
+
+
+class MockLims:
```

Comment:
Can you add this to limsmock in cg/tests/mocks/limsmock as extra functionality?

Reply:
Sure

Reply:
Fixed